src/mo_def/syntax.ml

```diff
@@ -235,7 +235,11 @@ and prog' = dec list
 (* Signatures (stable variables) *)
 
 type stab_sig = (stab_sig', prog_note) Source.annotated_phrase
-and stab_sig' = (dec list * typ_field list)      (* type declarations & stable actor fields *)
+and stab_sig' = (dec list * stab_body)      (* type declarations & stable actor fields *)
+and stab_body = stab_body' Source.phrase    (* type declarations & stable actor fields *)
+and stab_body' =
```

Comment:
one could have only `PrePost` and when both equal unparse it as version 1.0.0. Similarly parse that and set `pre` = `post`.

Reply:
I think its cleaner this way. PrePost means migration is involved, Single means just implicit. Also, it is not trivial to check equality.